src/applications/detection/main.cpp

```diff
@@ -77,16 +77,19 @@ void drawMarkers(const boost::ptr_list<CCTag> &markers, cv::Mat &image)
     if(marker.getStatus() == status::id_reliable)
     {
       const cv::Scalar color = cv::Scalar(0, 255, 0 , 255);
+        const auto rescaledOuterEllipse = marker.rescaledOuterEllipse();
+        std::cout << "a " << rescaledOuterEllipse.a() << " b " << rescaledOuterEllipse.b() << " angle " << rescaledOuterEllipse.angle() << std::endl;
```

Comment:
Do you want to keep the cout line for information about the ellipse?

Reply:
nope, it was just debugging stuff, i'll remove it.

Reply:
d31f2d0